charts/stable/fireflyiii/templates/_cronjob.tpl

```diff
@@ -0,0 +1,38 @@
+{{/* Define the cronjob */}}
+{{- define "fireflyiii.cronjob" -}}
+{{- $jobName := include "common.names.fullname" . }}
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ printf "%s-cronjob" $jobName }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+  schedule: "{{ .Values.cronjob.schedule }}"
+  concurrencyPolicy: Forbid
+  {{- with .Values.cronjob.failedJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ . }}
+  {{- end }}
+  {{- with .Values.cronjob.successfulJobsHistoryLimit }}
+  successfulJobsHistoryLimit: {{ . }}
+  {{- end }}
+  jobTemplate:
+    metadata:
+    spec:
+      template:
+        metadata:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: {{ .Chart.Name }}
+              image: radial/busyboxplus:curl
+              imagePullPolicy: {{ default .Values.image.pullPolicy }}
+              args:
+              - curl
+              - "http://{{ .Release.Name }}-fireflyiii.ix-{{ .Release.Name }}.svc.cluster.local:{{ .Values.service.main.ports.main.targetPort }}/api/v1/cron/{{ .Values.secret.STATIC_CRON_TOKEN }}"
```

Comment:
This is most-likely wrong.
The first portion IS the release name.

Reply:
Apps are called release.name, not release.name-chart.name

Reply:
Well if the user
- names the app `fireflyiii` the dns name would be
`fireflyiii.ix-fireflyiii.svc.cluster.local`
- but if user names it `ffiii` the dns would be
`ffiii-fireflyiii.ix-ffiii.svc.cluster.local`

Right?
I have to accommodate for both scenarios. Probably with an `if` but I need to dig on other examples on how it's syntaxes is.

Reply:
Did you test that?

Reply:
Don't asume, test.

Reply:
Also, a name generator for the full name is available just fine.
You're even already using it in the cronjob.